### PR TITLE
folder_block_ops: track added sym links in the cache

### PR DESCRIPTION
**Pull request by strib**
Issue: KBFS-2076

### CODE REVIEW

**Comment by strib** [comment]
> Will removed symlinks be captured in dels? What about ones that were added and removed in the same MD update? Presumably now you have to remove them from the addedSyms map rather than adding it to dels.

This PR does delete removed symlinks from `dels`: https://github.com/keybase/kbfs/pull/945/files#diff-ca9a77b130940ad6ae13c6cb2d59dcd8R888

**Comment by jzila** [comment]
Doh! Missed that somehow.

**Comment by strib** [comment]
(Don't know why appveyor tried to run on this non-master-branch PR, you can ignore this failure.)

File: libkbfs/folder_block_ops.go [hunk 1]
```diff
@@ -134,6 +134,8 @@ type deCacheEntry struct {
 	// dels is a set of the names that have been removed from the
 	// DirBlock for the BlockPointer that maps to this struct.
 	dels map[string]bool
+	// addedSyms is a map of the dir entries for added sym links.
```
Comment (akalin-keybase) [inline libkbfs/folder_block_ops.go]: ultra-nit: "sym links" -> "symlinks" (or "symbolic links")?

**Pull request merged**